database/scripts/seed.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+const Soundtrack = require('../models/soundtracks.js');
+
+mongoose.connect('mongodb://localhost/bonuses', {
+  autoIndex: false,
+  dbName: 'bonuses',
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+}).then( () => {
+  console.log('Connection to MongoDB successful!');
+}).catch( (err) => {
+  console.log('DB connection error:', err);
+});
+
+let albums  = [];
+
+const fakeSongList = () => {
+  let songs = [];
+  let listLength = Math.floor(Math.random() * 13) + 9;
+
+  for (let j = 0; j < listLength; j++) {
+    let number = j.toString();
+
+    if (j < 10) {
+      number = number.padStart(2, "0");
+    }
+
+    let song = {
+      track_number: number,
+      name: faker.lorem.words(),
+      song: faker.internet.url()
+    }
+    songs.push(song);
+  }
+  return songs;
+}
+
+for (let i = 1; i <= 10; i++) {
+  let additionalBonus = Math.floor(Math.random() * Math.floor(100));
```

Comment:
I don't think you need to `Math.floor(100)` if you're hardcoding `100` in. I also don't think you need `100` given what you're doing with `additionalBonus`, something like `Math.round(Math.random())` will give the same results without raising questions about where the 100 is coming from.

Reply:
1) The second Math.floor is a typo. Will update.
2) I prefer the distribution of 0s and 1s that I get having a number in there. It's a pattern with a higher velocity.
3) I like the number 100.